internal/integrations/github/github_test.go

```diff
@@ -0,0 +1,80 @@
+package github
```

Comment:
Feel like the test suite got a lot more confusing.

Reply:
I don't think it's too confusing. Defining the `github.ListCheckRunResults` values by hand is a bit verbose, but we only do that in one place. A `newCheckRun(detailsURL, app string) *github.CheckRun` test helper could help there, but for now I don't think it's necessary.